### ISSUE TITLE
Input event not firing in a scene launched in parallel

### ISSUE THREAD

**Issue by adrianolobo** [photonstorm/phaser#4327]
## Version
* Phaser Version: 3.16 RC 1

* Operating system: Linux Deepin and Android

## Description

When launching a scene in parallel `this.scene.launch('***')` above the current scene, like a modal, the input events on the newest scene doesn't fire.

## Example Test Code
`
class Scene1 extends Phaser.Scene {

    constructor (config)
    {
        super({ key: 'Scene1' });
    }

    create ()
    {
        this.overlay = this.add.rectangle(
            this.cameras.main.width/2,
            this.cameras.main.height/2,
            this.cameras.main.width,
            this.cameras.main.height,
            0xff0000
        );
        this.overlay.setInteractive();
        this.btn = this.add.rectangle(
            this.cameras.main.width/2,
            this.cameras.main.height/2,
            100,
            30,
            0xffffff
        );
        this.btn.setInteractive();
        this.btn.on('pointerdown', () => {
            this.scene.launch('Scene2')
        });
    }

}

class Scene2 extends Phaser.Scene {

    constructor (config)
    {
        super({ key: 'Scene2' });
    }

    create ()
    {
        this.overlay = this.add.rectangle(
            this.cameras.main.width/2,
            this.cameras.main.height/2,
            this.cameras.main.width,
            this.cameras.main.height,
            0x00ff00
        );
        this.overlay.setInteractive();
        this.btn = this.add.rectangle(
            this.cameras.main.width/2,
            this.cameras.main.height/2,
            100,
            30,
            0x0000ff
        );
        this.btn.setInteractive();
        this.btn.on('pointerdown', () => {
            alert('Scene 2 BTN')
        });
    }

}

var config = {
    type: Phaser.WEBGL,
    width: 800,
    height: 600,
    backgroundColor: '#000000',
    parent: 'phaser-example',
    scene: [Scene1, Scene2]
};

var game = new Phaser.Game(config);

`

**Comment by photonstorm** [comment]
Duplicate of #4321 - will look into it this week.

**Comment by photonstorm** [comment]
Re-opening, as the more eyes I get on this the better. I'm really struggling to re-create the issue. I've made a simple test case with 3 parallel Scene's and it works fine for me: http://labs.phaser.io/view.html?src=src/bugs/click%20scenes.js&v=dev

Does this work for you?

The only difference between mine and yours is I didn't use a Rectangle as the input. Although I'm going to try that now.

**Comment by photonstorm** [comment]
Ok, am narrowing this down.. if the hit objects overlap, the bottom one takes priority (instead of the top one)
